aptos-move/framework/aptos-stdlib/sources/cryptography/algebra_bls12381.move

```diff
@@ -17,28 +220,525 @@ module aptos_std::algebra_bls12381 {
     /// NOTE: the same scheme is also used in other implementations (e.g., ark-bls12-381-0.4.0, blst-0.3.7).
     struct FrFormatLsb {}
 
+    /// A serialization scheme for `Fr` elements,
+    /// where an element is represented by a byte array `b[]` of size 32 with the most significant byte coming first.
+    ///
+    /// NOTE: the same scheme is also used in other implementations (e.g., ark-bls12-381-0.4.0, blst-0.3.7).
+    struct FrFormatMsb {}
+
+    //
+    // (Marker types + serialization formats end here.)
+    // Hash-to-structure suites begin.
+    //
+
+    /// The hash-to-curve suite `BLS12381G1_XMD:SHA-256_SSWU_RO_` that hashes a byte array into `G1Affine` elements.
+    ///
+    /// Full specification is defined in https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-hash-to-curve-16#name-bls12-381-g1.
+    struct H2SSuiteBls12381g1XmdSha256SswuRo {}
+
+    /// The hash-to-curve suite `BLS12381G2_XMD:SHA-256_SSWU_RO_` that hashes a byte array into `G1Affine` elements.
+    ///
+    /// Full specification is defined in https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-hash-to-curve-16#name-bls12-381-g2.
+    struct H2SSuiteBls12381g2XmdSha256SswuRo {}
+
     //
-    // (Marker types and their serialization schemes end here).
+    // (Hash-to-structure suites end here.)
     // Tests begin.
     //
 
     #[test_only]
```

Comment:
Let's move these tests to their own `algebra_bls12381_tests.move` file.

Reply:
I think it's good to have test cases and the items being tested go together, in the same file, `ristretto255.move` being an example.

@alinush what's the concern?

Reply:
Maybe it's fine. I don't feel strongly. I just thought it'd be nice to have a clean file that presents the BLS constants (and, in the future, the inline functions) and another file that has all the tests.